### PR TITLE
docs: added the product listing page

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Product Listing page.

image

### CODE REVIEW

File: docs/prod-listing.html [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+                <span class="material-icons md-24 material-icons-outlined">
+                    favorite_border
+                </span>
+            </div>
+            <div class="card-child card-body">
+                <hr>
+                <p class="card-desc  p-sm">Ruled Journal - Hard Bound - A5, 100GSM, 160 Pages</p>
+            </div>
+            <div class="card-child card-footer">
+                <button class="btn link-btn-outline icon-dark">
```
Comment (swethaogeti) [inline docs/prod-listing.html:224]: this is good ! good naming practices

File: stylesheets/prod-listing.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+.products-badge:hover {
+    cursor: pointer;
+}
+
+.products-container .card-footer {
+    outline: none;
+    margin: auto;
+    margin-top: 0.5rem;
+}
```
Comment (swethaogeti) [inline stylesheets/prod-listing.css:69]: clean code ! everything looks good ! GTM

**Review by swethaogeti — APPROVED** [review]

**Review by kushalsourav — APPROVED** [review]
clean code , good to merge

**Comment by SinchanaMS** [comment]
Thank you for the review! 
Merging to dev.

**Pull request merged**